langroid/embedding_models/models.py

```diff
@@ -32,6 +32,17 @@ class SentenceTransformerEmbeddingsConfig(EmbeddingModelsConfig):
     devices: Optional[list[str]] = None
 
 
+class FastEmbedEmbeddingsConfig(EmbeddingModelsConfig):
+    model_type: str = "fastembed"
+    model_name: str = "BAAI/bge-small-en-v1.5"
+    dims: int = 384
```

Comment:
Having to declare the dims is not ideal, since the dims should just be a function of the model. Is there some way to get the dimensions from the model object? E.g. we do this with the HF models

Reply:
Have to run the inference once atleast.

Reply:
ok. It would be nice to add the SparseTextEmbedding and the LateInteractionTextEmbeddings as well (these are on the fastembed repo) -- is that possible?

Reply:
They return different types of results. Not `List[float]`. So they won't fit in this interface.

Reply:
Regarding the dimensions.
[`5bb7802` (#521)](https://github.com/langroid/langroid/pull/521/commits/5bb7802bd18c112fd769a01d3109b74e80d0932b)